root.tf

```diff
@@ -776,3 +776,183 @@ module "bastion_role" {
   name               = "BastionEC2Role${title(local.environment)}"
   policy_attachments = {}
 }
+
```

Comment:
This is getting quite large now. Is it possible to split this file up?

Reply:
Yeah, makes sense, I've moved the new Keycloak resources into a new file